src/wallet/wallet2.cpp

```diff
@@ -6338,10 +6338,10 @@ void wallet2::transfer_selected_rct(std::vector<cryptonote::tx_destination_entry
 			{
 				std::unordered_set<rct::key> new_used_L;
 				size_t src_idx = 0;
-				THROW_WALLET_EXCEPTION_IF(selected_transfers.size() != sources.size(), error::wallet_internal_error, "mismatched selected_transfers and sources sixes");
+				THROW_WALLET_EXCEPTION_IF(selected_transfers.size() != sources_copy.size(), error::wallet_internal_error, "mismatched selected_transfers and sources sixes");
```

Comment:
Note: This is not part of the original Monero PR but we need to check the size of `sources_copy` here because this is what we use inside of the loop.